pkg/namesgenerator/names-generator.go

```diff
@@ -374,6 +374,9 @@ var (
 		// Marie-Sophie Germain - French mathematician, physicist and philosopher. Known for her work on elasticity theory, number theory and philosophy. https://en.wikipedia.org/wiki/Sophie_Germain
 		"germain",
 
+		// Ruth Bader Ginsburg served as associate justice of the Supreme Court of the United States and spent much of her career as an advocate for gender equality and women's rights. https://en.wikipedia.org/wiki/Ruth_Bader_Ginsburg
```

Comment:
Does she belong to `notable scientists and hackers`?

Reply:
I would say she definitely "hacked the system" 😇